<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserTest</class>
 <widget class="QWidget" name="UserTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Use Signal Constellation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="AddArrowsByValue">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Add Arrows By Value</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QCheckBox" name="DarkTheme">
         <property name="text">
          <string>Dark</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="LightTheme">
         <property name="text">
          <string>Light</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="ConstellationWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>669</width>
           <height>500</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="ShowPointsCoord">
           <property name="text">
            <string>ShowPoints</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ShowArrowssCoord">
           <property name="text">
            <string>ShowArrowss</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConstellationWidget</class>
   <extends>QWidget</extends>
   <header>ConstellationWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
